dkist/dataset/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .dataset import Dataset
+from .loader import load_dataset
```

Comment:
can we bubble this up to `dkist.load_dataset` as well. Also are there not examples in the docs which will need updating?

Reply:
Thought I'd updated the docs, because the builds were failing. But I'll check again.